sdkman_release_new_candidate_version.sh

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
+
+# This script abstracts away API calls to sdkman's HTTP API for publishing new candidate versions.
+#
+# Example usage:
+# ./sdkman_release_new_candidate_version.sh 1.2.3
+
+set -o errexit
+set -o pipefail
+
+# Script starts here
+
+command -v curl >/dev/null 2>&1 || { echo "'curl' is required but not installed. Exiting." >&2; exit 1; }
+
+if [ "$#" -ne 1 ]; then
+    echo "Incorrect number of arguments. Usage: './sdkman_release_new_candidate_version.sh 1.2.3'. Exiting."
+    exit 1
+fi
+
+if [[ -z "${SDKMAN_CONSUMER_KEY}" ]]; then
+  echo "ENV var 'SDKMAN_CONSUMER_KEY' required but not set. Exiting."
+  exit 1
+fi
+
+if [[ -z "${SDKMAN_CONSUMER_TOKEN}" ]]; then
+  echo "ENV var 'SDKMAN_CONSUMER_TOKEN' required but not set. Exiting."
+  exit 1
+fi
```

Comment:
Where would we get these values if we wanted to use the script?

Reply:
those are secrets i've received at SDKMAN! vendor registration time. I've shared them with one or two Joern folk